Lib/typing.py

```diff
@@ -2457,6 +2457,19 @@ def __init__(self, name, tp):
         if def_mod != 'typing':
             self.__module__ = def_mod
 
+    def __mro_entries__(cls, bases):
+        new_type_name = cls.__name__
+
+        class Dummy:
+            def __init_subclass__(cls):
+                current_name = cls.__name__
+                raise TypeError(
+                    f'Cannot subclass {new_type_name}, perhaps you were looking for: '
+                    f'{current_name} = NewType({current_name!r}, {new_type_name})'
```

Comment:
This error message tells me _what_ to do instead, but doesn't tell me _why_ the error is being raised (it's an instance of `NewType`).

Reply:
```suggestion
                    f'Cannot subclass an instance of NewType, perhaps you were looking for: '
                    f'{current_name} = NewType({current_name!r}, {new_type_name})'
```

Reply:
That looks a lot better to me!